packages/design-system-tokens/src/lib/exportCsv.ts

```diff
@@ -0,0 +1,56 @@
+import { FileDescriptor } from './types';
+import { writeFile } from './utility';
+
+const tokenFormat = (t: string, k: string, v: string | unknown) => {
```

Comment:
What do each of these variables stand for? Can you update the variable names for clarity or add some function docs?